Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -236,6 +236,7 @@ def audit_style
       return unless @style_offenses
       @style_offenses.each do |offense|
         if offense.cop_name.start_with?("NewFormulaAudit")
+          next if formula.versioned_formula?
```

Comment:
Not sure if there is a better way to disable all `NewFormulaAudit` for @ formulae?